src/astro/databases/snowflake.py

```diff
@@ -63,7 +63,7 @@ def load_pandas_dataframe_to_table(
 
         db.prep_table(
             source_dataframe,
-            target_table.name,
+            target_table.name.lower(),
```

Comment:
A comment might help on why we lowercase it here

Reply:
I'll add it